client/me/purchases/payment-methods/payment-method-list.jsx

```diff
@@ -74,18 +76,19 @@ class PaymentMethodList extends Component {
 	}
 
 	render() {
+		let paymentMethods = this.props.cards;
+		if ( this.props.hasLoadedFromServer && this.props.paymentAgreements.length > 0 ) {
+			paymentMethods = paymentMethods.concat( this.props.paymentAgreements );
+		}
+
 		return (
 			<div className="payment-method-list">
 				<QueryStoredCards />
-				<SectionHeader label={ this.props.translate( 'Manage Your Payment Agreements' ) }>
+				<SectionHeader label={ this.props.translate( 'Manage Your Payment Methods' ) }>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Manage payment methods' )` *ES Score: 6*